packages/dd-trace/src/log.js

```diff
@@ -7,9 +7,29 @@ const _default = {
   error: err => console.error(err) /* eslint-disable-line no-console */
 }
 
+const _logLevels = {
```

Comment:
Let's use the same log levels (subset) as [bunyan](https://github.com/trentm/node-bunyan#levels) since they are easier to extend with new values.

Reply:
Updated, only added the currently accepted subset of bunyan values for now